cluster/photon-controller/util.sh

```diff
@@ -781,31 +789,87 @@ function verify-ssh-prereqs {
 function verify-photon-config {
     kube::log::status "Validating Photon configuration..."
 
-    fail=0
-    rc=0
+    # We don't want silent failure: we check for failure
+    set +o pipefail
+
+    verify-photon-flavors
+    verify-photon-image
+    verify-photon-tenant
+
```

Comment:
This verification happens very early so that users get errors quickly. The VM creation happens later, perhaps much later. So I really do want to do it up-front like this.